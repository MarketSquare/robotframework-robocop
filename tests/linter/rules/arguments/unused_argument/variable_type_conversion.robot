*** Keywords ***
Move
    [Arguments]    ${distance: int}    ${direction: Literal["UP", "DOWN"]}    ${slow: bool}=True
    No Operation

Move ${distance: int | float} meters
    No Operation
