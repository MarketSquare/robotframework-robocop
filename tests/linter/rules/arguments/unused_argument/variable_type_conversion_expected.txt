variable_type_conversion.robot:3:20 [W] ARG01 Keyword argument '${distance: int}' is not used
variable_type_conversion.robot:3:40 [W] ARG01 Keyword argument '${direction: Literal["UP", "DOWN"]}' is not used
variable_type_conversion.robot:3:79 [W] ARG01 Keyword argument '${slow: bool}' is not used
variable_type_conversion.robot:6:6 [W] ARG01 Keyword argument '${distance: int | float}' is not used